### PR TITLE
Instruments variable incorrectly defined

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
This was showing up as an error on my IDE, the variable is defined as private in the hchild class while the parent class refers to it.

change from `_select...` to `select...` because static checks enforce the ugly `_` before private variable rule

### CODE REVIEW

File: php/libraries/NDB_BVL_Instrument.class.inc [hunk 1]
```diff
@@ -182,6 +182,11 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     public $preview = false;
 
+    /**
+     * Array used by LINST instruments
```
Comment (driusan) [inline php/libraries/NDB_BVL_Instrument.class.inc]: Can you make the comment more descriptive? Used for what?

**Comment by ridz1208** [comment]
@lingma as per the description yes it is mandatory by our static analyses tools

@driusan I can but do you know what it's used for ? Or do I dig deeper ?

**Comment by lingma** [comment]
> @lingma as per the description yes it is mandatory by our static analyses tools

Then you need to modify several others files including at least 4 instruments, some wifi pages.

**Comment by ridz1208** [comment]
@lingma please request changes and specify which classes need to be modified. Thanks

**Comment by driusan** [comment]
It looks like it's a list of multi-select elements kept so that they can be converted to/from `{@}` delimited on save.

**Comment by johnsaigle** [comment]
I agee with Ling that the other files should be updated too. At least the Raisinbread instruments.

**Comment by ridz1208** [comment]
@lingma @johnsaigle does that cover everything ?

**Comment by driusan** [comment]
@ridz1208 the comment update is still missing

**Comment by lingma** [comment]
> @ridz1208 the comment update is still missing

Right, the `//` is missing. In fact `$this->selectMultipleElements = array();` or 
`//$this->selectMultipleElements = array();` should be the same.

**Comment by driusan** [comment]
No, I mean the update to the comment that I requested be updated 3 days ago because it's not very descriptive still isn't very descriptive.

**Comment by ridz1208** [comment]
@lingma i removed that comment on purpose, it's better to just declare the empty array

@driusan done

**Comment by driusan** [comment]
@ridz1208 Makes sense to me, look better now.

File: php/libraries/NDB_BVL_Instrument.class.inc [hunk 2]
```diff
@@ -182,6 +182,11 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     public $preview = false;
 
+    /**
+     * Array conaining all multiselct elements in an instrument.
```
Comment (lingma) [inline php/libraries/NDB_BVL_Instrument.class.inc]: containing, not "conaining"

Comment (ridz1208) [reply php/libraries/NDB_BVL_Instrument.class.inc]:
```suggestion
     * Array containing all multiselct elements in an instrument.
```

File: php/libraries/NDB_BVL_Instrument.class.inc [hunk 3]
```diff
@@ -182,6 +182,11 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     public $preview = false;
 
+    /**
+     * Array containing all multiselct elements in an instrument.
```
Comment (driusan) [inline php/libraries/NDB_BVL_Instrument.class.inc]:
```suggestion
     * Array containing all multiselect elements in an instrument.
```

**Pull request merged**